packages/frontend/src/pages/EventFormPage/EventFormPage.tsx

```diff
@@ -185,6 +272,14 @@ export const CreateEventPage = () => {
     volunteersQuantity,
   ]);
 
+  const formatDateString = (dateString: string): string => {
```

Comment:
please use getFormatDate from packages\frontend\src\components\formElements\DatePicker\utils\index.ts instead

Reply:
getFormatDate is accepting only Date objects so I have enhanced getFormatDate to handle both Date objects and string inputs from Firestore (e.g., '31 Aug 2024'), ensuring proper formatting across different date types.